R/broom_wrapper.R

```diff
@@ -508,10 +512,20 @@ prediction <- function(df, data = "training", data_frame = NULL, conf_int = 0.95
 
       # Use formula to support expanded aug_args (especially for type.predict for logistic regression)
       # because ... can't be passed to a function inside mutate directly.
-      aug_fml <- if(aug_args == ""){
-        "broom::augment(m, data = df)"
-      } else {
-        paste0("broom::augment(m, data = df, ", aug_args, ")")
+      if (!is.null(df$model[[1]]$prediction_training) || # Check if model already has training prediction result. # TODO: Make this model agnostic.
```

Comment:
Is `df$model[[1]` itself guaranteed not null?

Is "lm_exploratory" (or "glm_exploratory") guaranteed to show up always at the beginning of the vector?

Reply:
The expression works even if df$model[[1]] is NULL.
lm/glm_exploratory class is always set at the beginning of the vector.

Reply:
Thanks for your clarification. I confirmed it.